requirements.txt

```diff
@@ -4,3 +4,4 @@ jsonschema==4.23.0
 ops==2.17.1
 pydantic==2.10.3
 cosl==0.0.47
+interface_hacluster @ git+https://github.com/charmed-kubernetes/charm-interface-hacluster@main
```

Comment:
Will Renovate work with this kind of requirement?

Reply:
updated this to use a tag